src/commandline_options.hpp

```diff
@@ -239,6 +239,19 @@ class commandline_options
 	bool windowed;
 	/** True if --with-replay was given on the command line. Shows replay of the loaded file. */
 	bool with_replay;
+	/** True if --wnoconsole was given on the command line. Controls whether to write to a command prompt on Windows. */
```

Comment:
I think this should be phrased differently, perhaps more along the lines of "Prevents logs from being written to the console window if Wesnoth is launched from the command prompt on Windows". Not entirely happy with that either, but…